common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -10,20 +10,68 @@
 
 from xmodule.x_module import XModule, STUDENT_VIEW
 from xmodule.seq_module import SequenceDescriptor
-from xblock.fields import Scope, ReferenceList
+from xmodule.studio_editable import StudioEditableModule, StudioEditableDescriptor
 from xmodule.modulestore.exceptions import ItemNotFoundError
+from xmodule.validation import StudioValidation, StudioValidationMessage
+from xblock.fields import Scope, ReferenceList, String
+from xblock.fragment import Fragment
 
 
 log = logging.getLogger('edx.' + __name__)
 
+# Make '_' a no-op so we can scrape strings
+_ = lambda text: text
+
 
 class ConditionalFields(object):
     has_children = True
-    show_tag_list = ReferenceList(help="List of urls of children that are references to external modules", scope=Scope.content)
-    sources_list = ReferenceList(help="List of sources upon which this module is conditional", scope=Scope.content)
-
-
-class ConditionalModule(ConditionalFields, XModule):
+    display_name = String(
+        display_name=_("Display Name"),
+        help=_("This name appears in the horizontal navigation at the top of the page."),
+        scope=Scope.settings,
+        default=_('Conditional')
+    )
+
+    show_tag_list = ReferenceList(
+        help=_("List of urls of children that are references to external modules"),
+        scope=Scope.content
+    )
+
+    sources_list = ReferenceList(
+        display_name=_("Source Components"),
+        help=_("The location IDs of the components whose attributes are used to determine whether a learner is shown "
```

Comment:
@oksana-slu @andy-armstrong Do I understand correctly that more than one source component can be specified? How does a course team member enter more than one component? (e.g. separated by commas?)